drivers/sensors/Kconfig

```diff
@@ -760,6 +760,27 @@ config KXTJ9_I2C_BUS_SPEED
 
 endif # SENSORS_KXTJ9
 
+config SENSORS_L86_XXX
+	bool "Quectel L86-XXX GNSS support"
+	default n
+	depends on SERIAL
+	---help---
+		Enable driver support for the L86-XXX series of GNSS modules.
+
+config L86_XXX_BAUD
+	int "Quectel L86-XXX baud rate"
+	default 9600
+	depends on SENSORS_L86_XXX && SERIAL_TERMIOS
+	range 4800 115200
+	---help---
+		Supported values are: 4800, 9600, 14400, 19200, 38400, 57600 and 115200
+
+config L86_XXX_FIX_INT
```

Comment:
Add a help comment to indicate what this is (even just the description from the manufacturer datasheet). Also if there are any invalid options in the range (like how you did for baud).